CONTRIBUTING.md

```diff
@@ -143,6 +156,12 @@ Our build will fail if source code is not formatted according to that style.
 The main goal is to avoid extensive reformatting caused by different IDEs having different opinion
 about how things should be formatted by establishing.
 
+For Windows users, please set autocrlf to true and line endings will be converted to Unix style line endings when adding files to the repo:
+
+```bash
+git config --global core.autocrlf true
```

Comment:
I like to set it globally, but maybe we should just recommend setting it locally
```suggestion
git config core.autocrlf true
```